lib/common_test/src/Makefile

```diff
@@ -40,6 +40,9 @@ RELSYSDIR = $(RELEASE_PATH)/lib/common_test-$(VSN)
 # Target Specs
 # ----------------------------------------------------
 
+BEHAVIOUR_MODULES= \
+	ct_suite
```

Comment:
@IngelaAndin, shouldn't this be enough? I'm trying to understand which is the `Makefile` missing this file.

Reply:
Sorry yesterday was crazy. I was just trying to decipher the error messages in the checks below. I cloned your repo and could also build the docs. I will try including it in the test and see if the build errors below goes away.

Reply:
👍